src/index.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+const state = {
+  tempValue: 75,
+  color: 'orange',
+  landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+  skyscape: '',
+  background: "url('../assets/summer.jpg')",
+  city: 'Orlando',
+  lat: '',
+  lon: '',
+};
+
+// API calls
+const getLatLon = () => {
+  const inputCityElement = document.getElementById('inputCity').value;
+  axios
+    .get('http://127.0.0.1:5000/location', { params: { q: state.city } })
+    .then((response) => {
+      state.lat = response.data[0].lat;
+      state.lon = response.data[0].lon;
+      console.log(state.lat);
+      console.log(state.lon);
+      getRealTemp();
+    })
+    .catch((error) => {
+      console.log('Error!', error);
+    });
+};
+
+const getRealTemp = () => {
+  console.log(state.lat);
+  console.log(state.lon);
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: state.lat,
+        lon: state.lon,
+      },
+    })
+    .then((response) => {
+      console.log(response);
+      // const tempValueK = response.data.current.temp;
+      const tempValueK = response.data.current.temp;
+      const tempValueF = ((tempValueK - 273.15) * 9) / 5 + 32;
```

Comment:
Even though this is a pretty short piece of code, it would be nice if it was in a helper method so which would help with readability. 

You could even put lines 44-50 into a helper method, something like `formatRealTemp` and that helper method would call your other hypothetical helper called `convertKelvinToFarenheit`